packages/twitch-pubsub-client/src/PubSubClient.ts

```diff
@@ -154,6 +155,26 @@ Register one using:
 		return this.getUserListener(user).onModAction(channel, callback);
 	}
 
+	/**
+	 * Adds a listener for arbitrary/undocumented events to the client.
```

Comment:
Just a not that while I pledged to rename the functions to `custom`, the documentation can use a bit more involved terms, since people don't need to type these :)